avocado-plugins-vt.spec

```diff
@@ -6,7 +6,7 @@
 
 Summary: Avocado Virt Test Plugin
 Name: avocado-plugins-vt
-Version: 41.0
+Version: %{version}
```

Comment:
@ldoktor I tried aswell, `make rpm` fails at my env as well, now have reverted spec file change in this pull.